Automattic/jetpack
# Google Analytics: initialize tracking only if the feature is enabled

## Proposed changes:
* Only initialize Google Analytics tracking if the feature is enabled.

### Other information:


## Jetpack product discussion
pfwV0U-4g-p2
https://github.com/Automattic/vulcan/issues/308

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
The PR does not bring any functional changes and can't be tested by itself, so we're testing the existing behavior.

1. Connect Jetpack.
2. Go to "Jetpack -> Settings -> Traffic", find the "Google Analytics" card and purchase a plan.
3. If you're reusing a site, please deactivate the feature in Calypso ("Tools -> Marketing -> Traffic") and delete the `jetpack_wga` option if it already exists (`wp option delete jetpack_wga`).
4. Activate GA in Calypso ("Tools -> Marketing -> Traffic"), enter a measurement ID (e.g. "G-12345").
5. Check the WP option `wp option get jetpack_wga`, confirm it has both the measurement ID and `is_active: true`.
6. Go to the site's frontend, confirm the page's source code contains your measurement ID.
7. Deactivate GA in Calypso, confirm the tracking code no longer shows up on the site's frontend.
8. Check the WP option `wp option get jetpack_wga`, confirm `is_active` changed to `false`.
